discovery-provider/src/solana/audius_data_transaction_handlers.py

```diff
@@ -175,6 +176,14 @@ def handle_update_is_verified(
     pass
 
 
+class CreateTrackData(TypedDict):
```

Comment:
can this be more generic than CreateTrackData? since it's going to be shared for other entities

Reply:
(naming wise)